### PR TITLE
Replace `-f` and `--force` with `-y` and `--yes`

### PR DESCRIPTION
**Pull request by strickvl**
## Describe changes
I replaced `-f` and `--force` with `-y` and `--yes` everywhere in our codebase, docs, tests and examples.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

**Review by htahir1 — CHANGES_REQUESTED** [review]
Ok so I like this change a lot but tbh we should not just delete this option all of a sudden. Either we add a deprecated to all these functions using this pattern (https://stackoverflow.com/questions/2536307/decorators-in-the-python-standard-lib-deprecated-specifically) or we accept both flags in the function and give a warning when they pass in `-f` and then slowly phase it out. What do you think?

**Comment by strickvl** [comment]
@htahir1 you're totally right I forgot we had that conversation about gradual deprecation.

I think we should do the second of your two suggestions. I'll implement that on top of these changes.

**Comment by strickvl** [comment]
@bcdurak totally right. There were some things inside the `.github` older, but not inside `docker/`.

File: src/zenml/cli/secret.py [hunk 1]
```diff
@@ -308,16 +308,24 @@ def delete_secret_set(
...
     is_flag=True,
     help="Force the deletion of all secrets",
     type=click.BOOL,
 )
+@click.option(
+    "--force",
+    "-f",
+    "old_force",
+    is_flag=True,
+    help="DEPRECATED: Force the deletion of all secrets",
```
Comment (htahir1) [inline src/zenml/cli/secret.py:323]: I would say "Use `-y` instead"

**Review by htahir1 — CHANGES_REQUESTED** [review]
Really cool! Just one small change and then we're good to go. I would also communciate to @fa9r and @ayush714 that we need to make this update to ZenFiles

File: src/zenml/cli/secret.py (hunk 1 above)
Comment (strickvl) [reply src/zenml/cli/secret.py:323]: I've made PRs in the ZenBytes and ZenFiles repository with both @fa9r and @ayush714 added as reviewers, so they should be aware of this change.

Comment (strickvl) [reply src/zenml/cli/secret.py]: My latest commit adds this message as well.

**Review by htahir1 — APPROVED** [review]
LGTM!

**Review by bcdurak — APPROVED** [review]

**Pull request merged**